marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/runEmptyMapping.sjs

```diff
@@ -0,0 +1,35 @@
+const mapping = require("/data-hub/5/builtins/steps/mapping/entity-services/main.sjs");
+const test = require("/test/test-helper.xqy");
+
+let assertions = [];
+function runStructuredPropertyMapping () {
+  let doc = fn.head(xdmp.unquote(`{"id":1,"transactionDate":"12/7/2019","customer":{"firstName":"Oralia","lastName":"Dinesen","gender":"F"},"items":[{"name":"Voltsillam","price":2.0,"quantity":7},{"name":"Latlux","price":7.17,"quantity":10},{"name":"Biodex","price":5.01,"quantity":2},{"name":"Fixflex","price":8.77,"quantity":6},{"name":"Keylex","price":5.57,"quantity":3}]}`));
+  let result = mapping.main({uri: '/test.json', value: doc}, {
+    mapping: {name: 'OrdersMapping', version: 4},
+    outputFormat: 'json'
+  }).value.root;
+  let instance = result.envelope.instance;
+  assertions = assertions.concat([
+    test.assertEqual(null, instance.OrderType.customer),
+    test.assertEqual(null, instance.OrderType.items),
+    test.assertEqual("1", fn.string(instance.OrderType.id)),
+    test.assertEqual('2019-12-07', fn.string(instance.OrderType.purchaseDate))
+  ]);
+}
+
+function runSimplePropertyMapping() {
+  let doc = fn.head(xdmp.unquote(`{"id":1,"transactionDate":"12/7/2019","customer":{"firstName":"Oralia","lastName":"Dinesen","gender":"F"},"items":[{"name":"Voltsillam","price":2.0,"quantity":7},{"name":"Latlux","price":7.17,"quantity":10},{"name":"Biodex","price":5.01,"quantity":2},{"name":"Fixflex","price":8.77,"quantity":6},{"name":"Keylex","price":5.57,"quantity":3}]}`));
```

Comment:
Same as above, use the simplest input doc possible to verify the problem is fixed.